README.md

```diff
@@ -7,24 +7,26 @@ supported. Common operations for any libvirt driver should work.
 
 Several command line tools are also provided:
 
- - `virt-install`: Create new libvirt virtual machines
- - `virt-clone`: Duplicate existing libvirt virtual machines
- - `virt-xml`: Edit existing libvirt virtual machines/manipulate libvirt XML
+- `virt-install`: Create new libvirt virtual machines
+- `virt-clone`: Duplicate existing libvirt virtual machines
+- `virt-xml`: Edit existing libvirt virtual machines/manipulate libvirt XML
 
 For dependency info and installation instructions, see the
-[INSTALL.md](INSTALL.md) file. If you just want to quickly test the
-code from a git checkout, you can launch any of the commands like:
+[INSTALL.md](INSTALL.md) file.
+Once you have all the pre-requisite dependencies installed,
+if you just want to quickly test the code from a git checkout,
+you can launch any of the commands like:
```

Comment:
To me this feels like duplicating information. This paragraph points to INSTALL.md where you can find more details about dependencies.